src/services/lnd/index.ts

```diff
@@ -247,7 +247,7 @@ export const LndService = (): ILightningService | LightningServiceError => {
       return payment
     }
 
-    return new PaymentNotFoundError("Payment hash not found")
+    return new PaymentNotFoundError(JSON.stringify({ paymentHash, pubkey }))
```

Comment:
do we need to return this info in the error? is this not known by the caller? (in case we want to do some tracing)

Reply:
I can't remember the details now, but I do remember there was some strange edge-case somewhere where we didn't have this info and I needed it to show up in the error